include/dlaf/common/index2d.h

```diff
@@ -245,10 +245,9 @@ Index2D<IndexT, Tag> computeCoordsColMajor(std::ptrdiff_t linear_index,
                                            const Size2D<IndexT, Tag>& dims) noexcept {
   using dlaf::util::ptrdiff_t::mul;
 
-  DLAF_ASSERT_MODERATE(linear_index >= 0, "The linear index cannot be negative (",
-                       std::to_string(linear_index), ")");
-  DLAF_ASSERT_MODERATE(linear_index < mul(dims.rows(), dims.cols()), "Linear index ",
-                       std::to_string(linear_index), " does not fit into grid ", dims);
+  DLAF_ASSERT_MODERATE(linear_index >= 0, "The linear index cannot be negative!", linear_index);
+  DLAF_ASSERT_MODERATE(linear_index < mul(dims.rows(), dims.cols()),
+                       "Linear index does not fit into grid!", dims);
```

Comment:
Here maybe a `linear_index` is missing
```suggestion
                       "Linear index does not fit into grid!", linear_index, dims);
```